### PR TITLE
TChinese csv fix

### PR DESCRIPTION
**Pull request by NetherDragonTw**
nah i just do this new version lol

### CODE REVIEW

File: Resources/string.csv [hunk 1]
```diff
@@ -191,7 +191,7 @@ SpeedBoosterInfoLong,(Crewmates):Finishing all the tasks boosts the player speed
...
-SeerInfoLong,"(Crewmates):Every time players die, Seers can see kill flash.",(クルーメイト陣営):\nプレイヤーが死亡するごとに、シーアはキルフラッシュを見ることができる。,(船员阵营):\n每当玩家死亡时，灵媒将会看到击杀闪烁。,,"(Член Экипажа):Каждый раз когда игроки умирают, Seer может видеть вспышку убийства."
+SeerInfoLong,"(Crewmates):Every time players die, Seers can see kill flash.(Crewmates):Every time players die, Seers can see kill flash.",(クルーメイト陣営):\nプレイヤーが死亡するごとに、シーアはキルフラッシュを見ることができる。,(船员阵营):\n每当玩家死亡时，灵媒将会看到击杀闪烁。,"(船員):當有玩家死亡時,靈媒將看到閃光","(Член Экипажа):Каждый раз когда игроки умирают, Seer может видеть вспышку убийства."
```
Comment (soukunsandesu) [inline Resources/string.csv:194]: not require

Comment (NetherDragonTw) [reply Resources/string.csv:194]: why

Comment (soukunsandesu) [reply Resources/string.csv:194]: image
It's weird.

Comment (NetherDragonTw) [reply Resources/string.csv:194]: oh bruh i see it
let me update it

**Pull request closed without merging**